app/src/renderer/components/transactions/TmLiTransaction.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="tm-li-tx">
+    <div class="tm-li-tx__icon">
+      <img
+        :style="{ borderColor: color }"
+        src="~assets/images/cosmos-logo.png"
+      />
+    </div>
+    <div class="tm-li-tx__content">
+      <div class="tm-li-tx__content__left">
+        <div class="tm-li-tx__content__caption">
+          <p class="tm-li-tx__content__caption__title">
+            <slot name="caption"></slot>
+          </p>
+        </div>
+        <div class="tm-li-tx__content__information">
+          <slot name="details"></slot>
+        </div>
+      </div>
+      <div class="tm-li-tx__content__block">
+        <a :href="'https://explorecosmos.network/blocks/' + block"
```

Comment:
explorecosmos.network maybe should be passed as parameter so we can manipulate this via env || default

Reply:
Probably not that important given that will become a local link once we merge it